components/brave_adaptive_captcha/BUILD.gn

```diff
@@ -0,0 +1,24 @@
+source_set("brave_adaptive_captcha") {
+  sources = [
+    "brave_adaptive_captcha_delegate.h",
+    "brave_adaptive_captcha_service.cc",
+    "brave_adaptive_captcha_service.h",
+    "environment.h",
+    "get_adaptive_captcha_challenge.cc",
+    "get_adaptive_captcha_challenge.h",
+    "server_util.cc",
+    "server_util.h",
+  ]
+
+  deps = [
+    "//base",
+    "//brave/components/api_request_helper:api_request_helper",
+    "//brave/components/brave_adaptive_captcha/buildflags",
```

Comment:
buildflags are used inside the target that requires the buildflags? That doesn't seem right

Reply:
Removed.